Dockerfile

```diff
@@ -3,17 +3,15 @@ FROM centos:7
 # Install dependencies
 RUN yum -y install epel-release  && \
     yum -y install bash wget unzip \
-           pexpect python-daemon  bubblewrap gcc \
-           bzip2  openssh openssh-clients python2-psutil\
-           python36 python36-devel python36-setuptools\
-           nginx supervisor && \
-           localedef -c -i en_US -f UTF-8 en_US.UTF-8
-RUN easy_install-3.6 -d /usr/lib/python3.6/site-packages pip && \
-    ln -s /usr/lib/python3.6/site-packages/pip3 /usr/local/bin/pip3
+    pexpect python-daemon  bubblewrap gcc \
+    bzip2  openssh openssh-clients python2-psutil\
+    python36 python36-devel python36-setuptools\
+    nginx supervisor && \
+    localedef -c -i en_US -f UTF-8 en_US.UTF-8
```

Comment:
What is the indenting change for?

Reply:
After removing the installation of pip, the indentation seems to be more consistent in this way, in fact it was doing automatically by my visual studio editor. :-).
Taking a look to the raw file you will see that the look is great

Reply:
OK